packages/valory/skills/trader_abci/handlers.py

```diff
@@ -215,3 +228,11 @@ def _handle_get_static_file(
                 self._send_ok_request_response(http_msg, http_dialogue, index_html)
         except FileNotFoundError:
             self._handle_not_found(http_msg, http_dialogue)
+
+    def _handle_get_funds_status(
+        self, http_msg: HttpMessage, http_dialogue: HttpDialogue
+    ) -> None:
+        """Handle a fund status request."""
+        self._send_ok_request_response(
+            http_msg, http_dialogue, self.funds_status.get_response_body()
+        )
```

Comment:
Thinking out loud for future improvements:

- Is there any chance for the calculation to fail at some point?
- I would assume yes, so is there any way to identify these failures?
- Maybe we should identify them and not return ok response in these cases.